test/integration/downloadAndProcessFile.js

```diff
@@ -19,17 +21,26 @@ const SUPPLIER_HEADER = constants.HEADERS.SUPPLIER;
 const ODS_CODE_HEADER = constants.HEADERS.ODS_CODE;
 
 const testFilePath = `test/resources/${SIMPLE_FILE_NAME}`;
+const testFileServer = 'http://some.server';
+const testFileServerPath = '/test-pomi.csv';
+const testFileUrl = `${testFileServer}${testFileServerPath}`;
+
 
 describe('app', () => {
-  describe('processing of data from locally held file', () => {
+  describe('processing of data from URL (stubbed)', () => {
+    const stubbedData = fs.readFileSync(testFilePath);
+    nock(testFileServer)
+      .get(testFileServerPath)
+      .reply(200, stubbedData);
+
     before('delete files and run process', () => {
       if (fs.existsSync(`${OUTPUT_DIR}/${testFilePath}`)) {
```

Comment:
I've used fs.accessSync instead. The behaviour of the new function is to throw an error rather than return a value so have created a utility method in fsHelper